hack/local-up-master/openshift-apiserver-manifests/03_apiservice.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: v1
```

Comment:
@soltysh 
> Maybe a script to simplify this?

Not sure what you mean here, should we render these files instead?